tests/check/test_stream.c

```diff
@@ -192,6 +199,11 @@ test_stream_item_create_from_type(char type, uint8_t id, SignedVideoCodec codec)
       bu_data = codec == SV_CODEC_H264 ? p_nalu_h264
                                        : (codec == SV_CODEC_H265 ? p_nalu_h265 : invalid_av1);
       break;
+    case 'O':
+      bu_data = codec == SV_CODEC_H264 ? oms_sei_nalu_h264
+                                       : (codec == SV_CODEC_H265 ? oms_sei_nalu_h265 : invalid_av1);
```

Comment:
But now you're adding an invalid OBU.

Think of this as a camera where the user has turned on ONVIF Media Signing for an AV1 recording. Since AV1 is not supported by Media Signing, the stream will contain no SEIs at all. It will definitely not add some invalid OBUs (invalid_av1) or arbitrary SEI (sei_av1).

If I ask for "IPIOPIOP" like in the added test the AV1 stream would then look like "IPIPIP", right.
Is it possible to make sure you get that behavior?